README.md

```diff
@@ -540,6 +541,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**Brainf**](https://github.com/fredhappyface/android.brainf) <sup>**[[F-Droid](https://f-droid.org/app/com.fredhappyface.brainf)]**</sup>
 * [**ChemistryLab**](https://gitlab.com/asdoi/ChemistryLab)
 * [**phyphox**](https://github.com/phyphox/phyphox-android) <sup>**[[F-Droid](https://f-droid.org/app/de.rwth_aachen.phyphox)]**</sup>
+* [**Trivia You**](https://github.com/Bnyro/TriviaYou) <sup>**[[F-Droid](https://f-droid.org/app/com.bnyro.trivia)]**</sup>
```

Comment:
The source project has been archived and is inactive.

Reply:
Removed!